bots/github/__init__.py

```diff
@@ -300,30 +300,40 @@ class Checklist(object):
     def __init__(self, body=None):
         self.process(body or "")
 
+    def format(self, item, check):
+        status = ""
+        if isinstance(check, basestring):
+            status = check + ": "
+            check = False
+        return " * [{0}] {1}{2}".format(check and "x" or " ", status, item)
+
+    def parse(self, line):
+        check = item = None
+        stripped = line.strip()
+        if stripped.startswith("* [ ] ") or stripped.startswith("- [ ] "):
```

Comment:
I think something like this would be clearer:
```python
if stripped[:6] in ["* [ ] ", "- [ ] ", "* [x] ", "- [x] "]:
    item = stripped[6:].strip()
    check = stripped[3] == "x"
```